cypress-tests/cypress/e2e/happyPath/platform/ceTestcases/apps/appImportAndExport.cy.js

```diff
@@ -328,85 +373,189 @@ describe("App Import Functionality", () => {
+    data.constantsName = fake.firstName
+      .toLowerCase()
+      .replaceAll("[^A-Za-z]", "");
+    data.constantsValue = "dJ_8Q~BcaMPd";
+
+    cy.apiCreateApp(data.appName1);
+
     cy.visit("/");
+
     //add constants
+
     cy.get(commonSelectors.workspaceConstantsIcon).click();
     cy.get(workspaceConstantsSelectors.addNewConstantButton).click();
     cy.clearAndType(commonSelectors.nameInputFieldd, data.constName);
     cy.get(commonSelectors.valueInputFieldd).click();
     cy.clearAndType(commonSelectors.valueInputFieldd, data.constName);
     cy.get(workspaceConstantsSelectors.constantsType("global")).check();
     cy.get(workspaceConstantsSelectors.addConstantButton).click();
+
     cy.verifyToastMessage(
-        commonSelectors.toastMessage,
-        workspaceConstantsText.constantCreatedToast("Global")
+      commonSelectors.toastMessage,
+      workspaceConstantsText.constantCreatedToast("Global")
     );
-    // cy.get(commonSelectors.appHeaderLable).should("be.visible");
-    // cy.openapp(data.appName1);
     cy.get('[data-cy="icon-dashboard"]').click();
+
     navigateToAppEditor(data.appName1);
     cy.dragAndDropWidget("Text Input", 50, 50);
     cy.waitForAutoSave();
     cy.skipEditorPopover();
     cy.get('[data-cy="default-value-input-field"]').clearAndTypeOnCodeMirror(
-        `{{constants.${data.constName}}`
+      `{{constants.${data.constName}}`
     );
     cy.forceClickOnCanvas();
     cy.waitForAutoSave();
+
+    // add data sources
+
+    cy.backToApps();
+    cy.get(commonSelectors.globalDataSourceIcon).click();
+
+    selectAndAddDataSource(
+      "databases",
+      dataSourceText.postgreSQL,
+      data.dsName1
+    );
+
+    cy.intercept("GET", "api/v2/data_sources").as("datasource");
+    fillConnectionForm(
+      {
+        Host: Cypress.env("pg_host"),
+        Port: "5432",
+        "Database Name": Cypress.env("pg_user"),
+        Username: Cypress.env("pg_user"),
+        Password: Cypress.env("pg_password"),
+      },
+      ".form-switch"
+    );
+    cy.wait("@datasource");
+    cy.openApp(data.appName1);
+
+    pinInspector();
+
+    addQuery(
+      "table_preview",
+      `SELECT * FROM persons;`,
+      `cypress-${data.dsName1}-postgresql`
+    );
+
+    cy.get('[data-cy="query-preview-button"]').click();
+
     //add tjdb
+
     cy.wait(2000);
     cy.get('[data-cy="show-ds-popover-button"]').click();
-    
-   });
-  it.skip("Verify 'Export app' functionality of an application inside editor", () => {
+    cy.wait(1000);
+    cy.get('[data-cy="ds-tooljet database"]').click();
+    cy.get('[data-cy="query-preview-button"]').click();
+    cy.wait(2000);
+
+    //Export the app
+
+    cy.backToApps();
+    selectAppCardOption(
+      data.appName1,
+      commonSelectors.appCardOptions(commonText.exportAppOption)
+    );
+
+    clickOnExportButtonAndVerify(
+      exportAppModalText.exportSelectedVersion,
+      data.appName1
+    );
+
+    // Import same app and verify components
```

Comment:
- Import the app into a different workspace and update the test cases accordingly.
- Add test cases for importing and exporting an app that contains TJDB.